src/server/controllers/verifyEmail.js

```diff
@@ -0,0 +1,9 @@
+const { updateEmailVerification } = require('./../../database/db_update');
+
+module.exports = (req, res) => {
+
+ updateEmailVerification(req.params.username, req.params.token, (error, response) => {
+   if (error) { return error }; //TODO: handle error
```

Comment:
Okay, if you do not want to handle the error properly, just send the client a 500 with the error object.

The problem here is that the request will hang, since you are not responding. This makes pain during development
ie just write `res.status(500).send(error)`

This will stop you banging your head against the table and wasting time